composer.json

```diff
@@ -28,42 +27,43 @@
       "homepage": "https://github.com/browscap/browscap/graphs/contributors"
     }
   ],
+  "homepage": "https://github.com/browscap/browscap",
+  "support": {
+    "issues": "https://github.com/browscap/browscap/issues",
+    "source": "https://github.com/browscap/browscap"
+  },
   "require": {
-    "php": "^7.1.3",
+    "php": "^7.4.3",
     "ext-json": "*",
-    "beberlei/assert": "^3.3.0",
-    "ergebnis/json-normalizer": "^0.13.0 || ^1.0.2",
-    "mimmi20/json-class": "~2.0.16",
-    "mimmi20/ua-browser-type": "^7.0.15",
-    "mimmi20/ua-device-type": "^7.1.14",
-    "monolog/monolog": "^1.26.0 || ^2.2.0",
+    "beberlei/assert": "^3.3.1",
+    "ergebnis/json-normalizer": "^1.0.3",
+    "mimmi20/ua-browser-type": "^8.0.5",
+    "mimmi20/ua-device-type": "^8.0.5",
+    "monolog/monolog": "^2.3.5",
     "seld/jsonlint": "^1.8.3",
-    "symfony/console": "^4.4.18 || ^5.2.1",
-    "symfony/finder": "^4.4.18 || ^5.2.1",
-    "twig/twig": "^2.13.1 || ^3.2.1"
+    "symfony/console": "^v5.4.0",
+    "symfony/finder": "^v5.4.0",
+    "twig/twig": "^v3.3.4"
   },
   "require-dev": {
     "ext-zip": "*",
-    "browscap/browscap-php": "^4.3.0 || ^5.0.0",
+    "browscap/browscap-php": "dev-updates",
```

Comment:
Needs making stable before merge

Reply:
As soon as browscap/browscap-php#406 is released

Reply:
browscap-php 6.0.0 is now released: https://github.com/browscap/browscap-php/releases/tag/6.0.0